vpn-store/src/main/java/com/duckduckgo/mobile/android/vpn/store/VpnDatabase.kt

```diff
@@ -29,14 +29,13 @@ import org.threeten.bp.OffsetDateTime
 import org.threeten.bp.format.DateTimeFormatter
 
 @Database(
-    exportSchema = true, version = 23,
+    exportSchema = true, version = 24,
```

Comment:
Bumped the version to remove the `VpnPreferences` that is no longer used as we use harmony shared prefs for the always-on feature instead of the db table